src/mkvdate.c

```diff
@@ -52,7 +52,8 @@ int main(void) {
   fprintf(stderr, "Mdate :%ld\n", (long)time.tv_sec);
   fprintf(stderr, "Version: %s\n", ctime(&time.tv_sec));
 
-  printf("long MDate= %ld;\n", (long)time.tv_sec);
+  printf("#include <time.h>\n");
+  printf("time_t MDate= %ld;\n", time.tv_sec);
```

Comment:
../src/mkvdate.c:56:34: warning: format specifies type 'long' but the argument has type 'time_t' (aka 'int') [-Wformat]
  printf("time_t MDate= %ld;\n", time.tv_sec);
                        ~~~      ^~~~~~~~~~~
                        %d

Reply:
Ugh. I updated with a force push that should address that.